foghorn-edge/templates/cleanup-folder-structure-job.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: batch/v1
```

Comment:
License and Copyright information is missing.
open-ness/edgeapps repo hosts the open source edge applications. The license must be open source License.